Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.39694&attempt_id=c1b650b3-7317-49d2-b49a-a188c4ad2040)

Attempted: python2Packages.pyslurm, python3Packages.pyslurm

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/9ib9731m2av0dxpnj36gdwr48c6qmzvd-python2.7-numpy-1.14.2/lib/python2.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it by " \
  ^~~~~~~
acquired default
acquired Global_A
building of '/nix/store/kh3qnjl7ik5ap4awv41hx6inl5k390sj-python2.7-Cython-0.28.1.drv' timed out after 3600 seconds
building of '/nix/store/dqfscwhjqfxd73arwp1gxf1y2wh0k6dm-python3.6-Cython-0.28.1.drv' timed out after 3600 seconds
cannot build derivation '/nix/store/cvjl5qgsbayd999zx3qr2rnhmwqj122p-python2.7-pyslurm-20180427.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/mz1w149idyg0yx35p7zif635rmxq8qsh-python3.6-pyslurm-20180427.drv': 1 dependencies couldn't be built
[31;1merror:[0m build of '/nix/store/cvjl5qgsbayd999zx3qr2rnhmwqj122p-python2.7-pyslurm-20180427.drv', '/nix/store/mz1w149idyg0yx35p7zif635rmxq8qsh-python3.6-pyslurm-20180427.drv' failed
```
</p></details>

--- a/pkgs/development/python-modules/pyslurm/default.nix
+++ b/pkgs/development/python-modules/pyslurm/default.nix
@@ -2,13 +2,13 @@
 
 buildPythonPackage rec {
   pname = "pyslurm";
-  version = "20170302";
+  version = "20180427";
 
   src = fetchFromGitHub {
     repo = "pyslurm";
     owner = "PySlurm";
-    rev = "f5a756f199da404ec73cb7fcd7f04ec4d21ea3ff";
-    sha256 = "1xn321nc8i8zmngh537j6lnng1rhdp460qx4skvh9daz5h9nxznx";
+    rev = "3900e1afac9ffd13c80c57d8c39933d42eb7bad7";
+    sha256 = "1a183ig4sdbc70rx2yyaslyq61wkbsf8cbim1jj0kzrp65nf0vls";
   };
 
   buildInputs = [ cython slurm ];

